### PR TITLE
readme: Fix the required version of runc

### PR DESCRIPTION
**Pull request by mrunalp**
Signed-off-by: Mrunal Patel <mrunalp@gmail.com>

### CODE REVIEW

**Comment by runcom** [comment]
LGTM

**Comment by runcom** [comment]
Mmm wait, not sure Rc3 works properly cause it doesn't have the "delete a container with force and gi

**Comment by rhatdan** [comment]
Shouldn't this be .6,  lets tell people to use the latest version.

**Comment by runcom** [comment]
> Shouldn't this be .6, lets tell people to use the latest version.

you let's just tell them to use latest at at least a minimum commit - when runc 1.0 is out we won't have this issue anymore

**Comment by mrunalp** [comment]
Changed to latest version.

File: README.md [hunk 1]
```diff
@@ -46,7 +46,7 @@ For sync communication we have an IRC channel #cri-o, on chat.freenode.net, that
 
 ### Prerequisites
 
-`runc` version 1.0.0.rc1 or greater is expected to be installed on the system. It is picked up as the default runtime by crio.
+Lastest verion of `runc` version is expected to be installed on the system. It is picked up as the default runtime by crio.
```
Comment (runcom) [inline README.md]: Lastest/Latest typo + version repeated twice

**Comment by TomSweeneyRedHat** [comment]
LGTM

**Comment by rhatdan** [comment]
LGTM

**Pull request merged**